### ISSUE TITLE
P1.py

### ISSUE THREAD

**Issue by benefoon** [benefoon/Credit-Risk-Prediction-System-Using-Machine-Learning#1]
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler

# Load dataset
data = pd.read_csv("credit_data.csv")

# Handle missing values
data.fillna(data.mean(), inplace=True)

# Encode categorical variables (if any)
data = pd.get_dummies(data, drop_first=True)

# Split data into features and labels
X = data.drop("target", axis=1)  # Replace "target" with your label column
y = data["target"]

# Train-test split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Standardize features
scaler = StandardScaler()
X_train = scaler.fit_transform(X_train)
X_test = scaler.transform(X_test)
